### PR TITLE
Download files

### PR DESCRIPTION
**Pull request by matt2695**
Added files downloads

### CODE REVIEW

File: src/lib/DatabaseGenerator.ts [hunk 1]
```diff
@@ -27,8 +27,13 @@ export const DatabaseGenerator = async (options: any) => {
 ${postfix}`
 
             await fs.writeFile(path.join(prismaStorage, 'schema.prisma'), sqlitePrisma);
-            await run(`${prismaExec} generate --schema "${prismaSchema}"`);
-            await run(`${prismaExec} migrate deploy --schema "${prismaSchema}"`);
+            try {
```
Comment (nabeards) [inline src/lib/DatabaseGenerator.ts:30]: I say wrap the entire `if` statement in a try/catch, and then provide a better console.error message, like `Unable to prepare ${options.dbProvider} database: ${e.message}`

**Review by nabeards — CHANGES_REQUESTED** [review]
Left a comment about the try/catch in DatabaseGenerator.ts
Also just need to run `npm run patch-version` to bump and tag it now.

File: src/lib/DatabaseGenerator.ts (hunk 1 above)
Comment (matt2695) [reply src/lib/DatabaseGenerator.ts:14]: updated take a look

**Comment by nabeards** [comment]
Will you be opening a new PR with this and the changes you mentioned to load small files into memory?

**Pull request closed without merging**